test/e2e_node/gpu_device_plugin_test.go

```diff
@@ -21,18 +21,48 @@ import (
 	"strconv"
 	"time"
 
+	"k8s.io/kubernetes/test/e2e/framework/gpu"
```

Comment:
You don't need to move this line from line 28 at all.

Reply:
@oomichi thanks a lot! fixed, could you check again?